My-Spec-Tutorial.package/MyFirstWindow.class/README.md

```diff
@@ -0,0 +1 @@
+MyFirstWindow open
```

Comment:
this is my mistake. Next time i will take care of adding comments telling about package and how to use it. Thanks.